frontend/src/AutocompleteWrapper.ts

```diff
@@ -43,9 +43,11 @@ class AutocompleteWrapper {
     }
 
     let detachedMediaQuery = undefined;
```

Comment:
you could just put by default wdyt?
```suggestion
    let detachedMediaQuery = 'none';
```

Reply:
In this case the default value would become `false`, the idea was to keep the Autocomplete.js default if nothing is set.

Reply:
we don't have to stick to autocomplete api 1:1 but why not